ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml

```diff
@@ -0,0 +1,145 @@
+base_images:
+  metrics-linter:
+    name: prom-metrics-linter
+    namespace: ci
+    tag: v0.0.2
```

Comment:
Does this ever have to be updated?

Reply:
Maybe @assafad will know? It was added in this PR: https://github.com/openshift/release/pull/41234

Reply:
I see Assaf doesn't work on this project anymore. Maybe @machadovilaca or @sradco would know?